graph/hwgraph.py

```diff
@@ -464,6 +465,14 @@ def main():
     # Call the appropriate sub command
     args.func(args)
 
+    # Save the actual cmdline so it's easier to regenerate graph
+    # Do not expose the actual hwbench path like /home/user/hwbench/...
+    sys.argv[0] = "hwgraph"
+    # Keep quotes from the argument list with shlex
+    pathlib.Path(pathlib.Path(args.outdir).joinpath("cmdline")).write_text(
+        " ".join(shlex.quote(arg) if " " in arg else arg for arg in sys.argv)
```

Comment:
I'm not sure this works as we expect. For example, shlex is only called for " ", but "\t" could be a space character that. My advice:
 - (preferred) run shlex.quote() for every argument (remove if " "); it's cheap, and it won't modify arguments that don't need quoting.
 - use json or a container that can separate arguments
 - use '\0' as a separator between arguments, but just like json it needs a tool for re-executing.